packages/studio-ui/src/web/components/Layout/TopNav/RightToolbar.tsx

```diff
@@ -0,0 +1,70 @@
+import { Icon, Tooltip } from '@blueprintjs/core'
+import { lang, ShortcutLabel, utils } from 'botpress/shared'
+import classNames from 'classnames'
+import React, { FC } from 'react'
+import { connect } from 'react-redux'
+import { RootReducer } from '~/reducers'
+
+import style from './style.scss'
+
+interface Props {
+  docHints: any[]
+  isEmulatorOpen: boolean
+}
+
+const RightToolBar: FC<Props> = props => {
```

Comment:
No change necessary but worth mentioning- people generally don't use `FC` anymore. You would just do `const RightToolBar = (props: Props) => ...`
https://github.com/facebook/create-react-app/pull/8177
https://www.harrymt.com/blog/2020/05/20/react-typescript-react-fc.html
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/